.github/pull_request_template.md

```diff
@@ -0,0 +1,69 @@
+# [AAP-XXXXX] Brief Title Describing the Change
+
+## Description
+<!-- Mandatory: Provide a clear, concise description of the changes and their purpose -->
+- What is being changed?
+- Why is this change needed?
+- How does this change address the issue?
+
+Closes #XXX (if applicable)
+Jira: [AAP-XXXXX](https://jira.aap.com/browse/AAP-XXXXX) (if applicable)
+
+## Type of Change
```

Comment:
ditto title

Reply:
Sorry, I do not follow you. What do you mean?